app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  def show
+    respond_to do |format|
+      format.html do
+        @ticket_id = params.require(:id)
+        render :index
```

Comment:
In line with my other comment below, this `index` view should probably be renamed `show` to match the controller route.